### ISSUE TITLE
[gateway.transport.wseb]: server send NOP command frame as soon as the downstream is attached

### ISSUE THREAD

**Issue by pkhanal** [kaazing/gateway#274]
While the behavior does not have any practical consequences, it is causing the spec test to fail as the behavior is not part of the WSE spec.

**Comment by cmebarrow** [comment]
We currently have a system property, "com.kaazing.gateway.server.transport.wse.SPECIFICATION", that can be set to "true" to turn on spec compliance which turns off sending of this NOOP frame so the spec tests can pass. The purpose of this bug would be to permanently remove the sending of the initial NOOP frame (i.e. essentially behave as if the property is always set to true). But before we can do this we would need to validate that ALL of our current supported clients still function correctly (when connecting via wse or wse+ssl) when the property is set to true.
